dynamic-config/model/src/test/resources/tc-config.cfg

```
@@ -0,0 +1,82 @@
+#
+# Copyright Terracotta, Inc.
+# Copyright IBM Corp. 2024, 2025
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cluster-name=mycluster
+# comment
+stripe-names=stripeA
+stripeA:node-names=A,B,C,D
+
+tc-properties= \
+\
+l2.healthcheck.l2.ping.enabled:true,\
+# comment1
+# comment2\
+l2.healthcheck.l2.ping.idletime : 5000,\
+l2.healthcheck.l2.ping.interval: 1000,\   
+l2.healthcheck.l2.ping.probes :3,\
+\
+
+l2.healthcheck.l2.socketConnect  :  true,\
+l2.healthcheck.l2.socketConnectTimeout:2,      \
+l2.healthcheck.l2.socketConnectCount:5,\
+#\
+l2.healthcheck.l1.ping.enabled:true,\
+l2.healthcheck.l1.ping.idletime:5000,       \
+l2.healthcheck.l1.ping.interval:1000,\
+# comment3\
+l2.healthcheck.l1.ping.probes:3,\
+l2.healthcheck.l1.socketConnect:true ,\
+l2.healthcheck.l1.socketConnectTimeout:2,\
+l2.healthcheck.l1.socketConnectCount:5,\
```

```suggestion
        \
        # comment3 \
        l2.healthcheck.l1.ping.probes:3, \
        l2.healthcheck.l1.socketConnect:true , \
        l2.healthcheck.l1.socketConnectTimeout:2, \
        l2.healthcheck.l1.socketConnectCount:5, \
```

Usually continued lines are indented and the backslash is separated from the previous characters by a space